### DISCUSSION TITLE
How can I use the fallback SMS number to regain access to my GitHub account.

### DISCUSSION THREAD

**Discussion by shashank-garg-robomq** [community/community#39222] [Other Feature Feedback, Questions, & Ideas]
### Account Access

Question

### Body

My System crashed and coincidently, I switched to a new phone.
I do not have access to:
1. Recovery code
2. SSH key/Device/PAT.

I remember adding my Fallback SMS number on my GitHub account. How can I use that number to receive an OTP? I did not find any option on UI.

**Comment by Preetiraj3697** [comment]
Yes, Go to this page https://docs.github.com/en/authentication/securing-your-account-with-two-factor-authentication-2fa/recovering-your-account-if-you-lose-your-2fa-credentials
All Process are already clear step by step.
